libs/datalogger/datalogger.ts

```diff
@@ -1,3 +1,12 @@
+enum LogSeparator {
+    //% block="tab"
+    Tab,
```

Comment:
Do we support enums as strings or use the ASCII value and use String.fromCharCode